package.json

```diff
@@ -118,10 +118,9 @@
 		"ansi-regex": "^5.0.1",
 		"minimist": "^1.2.7"
 	},
-	"prettier": "@sapphire/prettier-config",
 	"packageManager": "yarn@3.3.0",
 	"dependencies": {
 		"fast-deep-equal": "^3.1.3",
-		"lodash.uniqwith": "^4.5.0"
+		"lodash": "^4.17.21"
```

Comment:
Let's not include the entirety of lodash please, just the parts we need (even if they're removed from the bundle, lodash is massive)